src/components/shared/CompactableListWithRemoveButton.css

```diff
@@ -0,0 +1,88 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.listWithRemoveButton {
+  z-index: 100;
```

Comment:
My rationale was:
1. the value can be very very big, so there is no particular reason to keep it short. You won't have a shortage of numbers ;)
2. we can use steps of 100 (or something else) between 2 components, so that inside one component we can use steps of 1 without looking elsewhere if we break something. For example I think Tooltips and ArrowPanels (for example) should likely get the same value.